spec/unit/virtus/attribute/boolean/coerce_spec.rb

```diff
@@ -40,4 +40,20 @@
       end
     end
   end
+
+  context 'when nullify_blank is turned on' do
+    let(:options) { { :nullify_blank => true }}
+
+    context 'with a coercible input' do
+      let(:input) { 1 }
+
+      it { is_expected.to be(true) }
+    end
+
+    context 'with a blank input' do
+      let(:input) { '' }
+
+      it { is_expected.to be_nil }
+    end
+  end
 end
```

Comment:
`nullify_blank` because it makes `strict` not to raise in specific cases, which I believe was one of the reasons why we wanted that feature :)